<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewerDlg</class>
 <widget class="QDialog" name="ViewerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>404</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1011</width>
    <height>404</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NSIM Demographics Viewer :: Details</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>991</width>
         <height>351</height>
        </size>
       </property>
       <property name="title">
        <string>Registration Details</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="8" column="3" colspan="2">
           <widget class="QLineEdit" name="txtStateOfOrigin">
            <property name="minimumSize">
             <size>
              <width>138</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>FEP Tracking ID:</string>
            </property>
            <property name="buddy">
             <cstring>txtFEPTrackingID</cstring>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>State of Origin:</string>
            </property>
            <property name="buddy">
             <cstring>txtStateOfOrigin</cstring>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Main Mobile Number:</string>
            </property>
            <property name="buddy">
             <cstring>txtMainMobileNumber</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="3" colspan="2">
           <widget class="QLineEdit" name="txtFirstName">
            <property name="minimumSize">
             <size>
              <width>277</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Registration Date:</string>
            </property>
            <property name="buddy">
             <cstring>txtRegistrationDate</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="5" rowspan="5">
           <widget class="QLabel" name="lblPortraitData">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>120</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>120</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="resources.qrc">:/images/person.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="3" colspan="2">
           <widget class="QLineEdit" name="txtResidentialAddress">
            <property name="minimumSize">
             <size>
              <width>277</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="txtFolioVersion">
            <property name="minimumSize">
             <size>
              <width>277</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QLineEdit" name="txtNationality">
            <property name="minimumSize">
             <size>
              <width>138</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Registration Type:</string>
            </property>
            <property name="buddy">
             <cstring>txtRegistrationType</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Mother's Maiden Name:</string>
            </property>
            <property name="buddy">
             <cstring>txtMotherMaidenName</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Folio Version:</string>
            </property>
            <property name="buddy">
             <cstring>txtFolioVersion</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="txtRegistrationType">
            <property name="minimumSize">
             <size>
              <width>277</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="3" colspan="2">
           <widget class="QLineEdit" name="txtMotherMaidenName">
            <property name="minimumSize">
             <size>
              <width>277</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Gender:</string>
            </property>
            <property name="buddy">
             <cstring>txtGender</cstring>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Residential Address State:</string>
            </property>
            <property name="buddy">
             <cstring>txtResidentialAddressState</cstring>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="txtRegistrationState">
            <property name="minimumSize">
             <size>
              <width>277</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="txtFEPTrackingID">
            <property name="minimumSize">
             <size>
              <width>277</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="3" colspan="2">
           <widget class="QLineEdit" name="txtSurname">
            <property name="minimumSize">
             <size>
              <width>277</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Surname:</string>
            </property>
            <property name="buddy">
             <cstring>txtSurname</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>First Name:</string>
            </property>
            <property name="buddy">
             <cstring>txtFirstName</cstring>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="txtRegistrationLGA">
            <property name="minimumSize">
             <size>
              <width>277</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QLineEdit" name="txtMainMobileNumber">
            <property name="minimumSize">
             <size>
              <width>138</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="3" colspan="2">
           <widget class="QLineEdit" name="txtResidentialAddressLGA">
            <property name="minimumSize">
             <size>
              <width>138</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="3" colspan="2">
           <widget class="QLineEdit" name="txtResidentialAddressState">
            <property name="minimumSize">
             <size>
              <width>138</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="txtRegistrationDate">
            <property name="minimumSize">
             <size>
              <width>277</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Subscriber ID:</string>
            </property>
            <property name="buddy">
             <cstring>txtSubscriberID</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="txtSubscriberID">
            <property name="minimumSize">
             <size>
              <width>277</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Date of Birth:</string>
            </property>
            <property name="buddy">
             <cstring>txtDateOfBirth</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="3" colspan="2">
           <widget class="QLineEdit" name="txtGender">
            <property name="minimumSize">
             <size>
              <width>277</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Residential Address LGA:</string>
            </property>
            <property name="buddy">
             <cstring>txtResidentialAddressLGA</cstring>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="txtRegistrationTime">
            <property name="minimumSize">
             <size>
              <width>277</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="3" colspan="2">
           <widget class="QLineEdit" name="txtDateOfBirth">
            <property name="minimumSize">
             <size>
              <width>277</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Registration Time:</string>
            </property>
            <property name="buddy">
             <cstring>txtRegistrationTime</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Residential Address:</string>
            </property>
            <property name="buddy">
             <cstring>txtResidentialAddress</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="txtFepCode">
            <property name="minimumSize">
             <size>
              <width>277</width>
              <height>0</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>FEP Code:</string>
            </property>
            <property name="buddy">
             <cstring>txtFepCode</cstring>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Nationality:</string>
            </property>
            <property name="buddy">
             <cstring>txtNationality</cstring>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Registration State:</string>
            </property>
            <property name="buddy">
             <cstring>txtRegistrationState</cstring>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Registration LGA:</string>
            </property>
            <property name="buddy">
             <cstring>txtRegistrationLGA</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>718</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnPrint">
         <property name="text">
          <string>&amp;Print</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnClose">
         <property name="text">
          <string>&amp;Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
  <include location="../../../../Python33/Lib/site-packages/PyQt5/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>ViewerDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>962</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
